### DISCUSSION TITLE
"?." and "??" gives Unexpected token error

### DISCUSSION THREAD

**Discussion by prabodhana** [nuxt/nuxt#9752] [Questions (Nuxt 2)]
I'm triyng to use pdfjs-dist package  [github link](https://github.com/mozilla/pdfjs-dist) my application. After install it give Unexpected token  error for if that package file wrote "?." or "??" . How can i fix this?

Page
```
<template>
  <div>
       <div id="pageContainer">
    <div id="viewer" class="pdfViewer"></div>
  </div>
  </div>
</template>

<script>
import pdfjsLib from "pdfjs-dist/build/pdf";
import { PDFViewer } from "pdfjs-dist/web/pdf_viewer";
import "pdfjs-dist/web/pdf_viewer.css";
pdfjsLib.GlobalWorkerOptions.workerSrc =
  "https://cdn.jsdelivr.net/npm/pdfjs-dist@2.0.943/build/pdf.worker.min.js";

export default {
    
    mounted() {
    this.getPdf();
  },
    methods:{
        async getPdf() {

        const container = document.getElementById("pageContainer");
        const pdfViewer = new PDFViewer({
            container
        });
        const loadingTask = pdfjsLib.getDocument("./pdf-sample.pdf");
        const pdf = await loadingTask.promise;
        pdfViewer.setDocument(pdf);
    }
}
}
</script>

<style>
#pageContainer {
  margin: auto;
  width: 80%;
}
</style>
```
Error Image:
![Screenshot from 2021-09-03 12-40-14](https://user-images.githubusercontent.com/39000060/131965178-8c1ef0dd-72a9-4d80-8d81-e260847382a5.png)

Thank you

**Comment by kissu** [comment]
Please share some code.

**Reply by prabodhana** [reply]
Thanks for the response. I'm updated the discussion with code and error.

**Comment by danielroe** [comment]
You should add any packages that use that syntax to your `build.transpile` array in your `nuxt.config` so they will be processed by Babel on build.
